### ISSUE TITLE
GUI: not all the widgets in a tool bar are visible in the corresponding context menu

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#2395]
The issue can be observed in:
 - [x] the `PMR` window;
 - [x] the `PMR Workspace` window;
 - [x] the `Web Browser` window; and
 - [x] the `Simulation Experiment` view.

For the latter, the tool bar looks as follows:

![Screen Shot 2020-09-08 at 18 20 18](https://user-images.githubusercontent.com/602265/92440142-256c5580-f200-11ea-9fd8-e55951b17611.png)

However, if you shrink the width of the application, you may end up with something like:

![Screen Shot 2020-09-08 at 18 20 37](https://user-images.githubusercontent.com/602265/92440192-39b05280-f200-11ea-894b-7eca2e0805af.png)

In other words, we can't see the whole contents of the tool bar, but there is the `>>` button at the far right of the too bar which can be used to show a context menu with the rest of the tool bar. However, here, our tool bar has some widgets (as opposed to some "simple" actions) and they cannot be seen in the context menu (e.g. the wheel, the drop down buttons):

![Screen Shot 2020-09-08 at 18 20 40](https://user-images.githubusercontent.com/602265/92440387-872cbf80-f200-11ea-9705-fc77709f8e10.png)
